assets/stylesheets/discourse_spoiler_alert.scss

```diff
@@ -14,7 +14,10 @@
 .spoiler-blurred {
   @include unselectable;
   cursor: pointer;
-  filter: blur(0.5em);
+
+  > * {
```

Comment:
I think this CSS rule is what is making it so the text is unblurred by default, if I change it back to the old one it works

Reply:
Ah interesting, it was working for me, but now when I check it out it doesn't work in all cases. Seems to be related to how certain CSS properties handle the child inheritance declaration.

I've updated the PR to remove that CSS change and am instead using the making the `:focus` state show the reduced blur similar to how it looks when we `:hover`

Let me know how it's looking now for you :)